pkg/controller/namespace/namespace_controller_utils.go

```diff
@@ -244,6 +244,9 @@ func syncNamespace(kubeClient clientset.Interface, versions *unversioned.APIVers
 	// we have removed content, so mark it finalized by us
 	result, err := retryOnConflictError(kubeClient, namespace, finalizeNamespaceFunc)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
Oh yeah - sure, sorry. Somehow I though it's doing "deleteContent" here, not "finalizing".
Yes - it looks ok. Thanks for explanation.

Maybe it's wort adding a comment, that it's mostly about protecting from two controllers doing their work?